conference/react-js/src/components/ForgotPassword/forgot-password-error-text.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import actions from '../../constants/actions-types';
+
+const ErrorText = ({ data }) => {
+  let res;
```

Comment:
You are no need res function and else statement at all.
You can rewrite function's body as: 
```
({data}) => {
  if (data === actions.EMAIL_IS_EMPTY) {
   return (
     <span className="field-error error-title_pop-up">
         Please enter your registered email
     </span>
    );
   }
    if (data === actions.EMAIL_NOT_FOUND) {
       return (
          <span className="field-error field-error_show error-title_pop-up">
             We can not find an account with that email address
          </span>
     );
   }
   return null}
```